libs/quasselc/Makefile

```diff
@@ -0,0 +1,71 @@
+#
+# Copyright (C) 2016 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=quasselc
+
+# quasselc upstream doesn't release versions (at least, at the moment),
+# so use 0 here (in case they decide to later) for now.
+# I'm not sure if there's an openwrt convention here.
+PKG_VERSION:=0
+PKG_SOURCE_VERSION:=fcd966966924e3d9af0954db56117e2f48767ea1
+PKG_RELEASE:=1.$(PKG_SOURCE_VERSION)
+
+PKG_LICENSE:=GPL-3.0+
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/phhusson/QuasselC
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
+
+PKG_MAINTAINER:=Ben Rosser <rosser.bjr@gmail.com>
+
+PKG_BUILD_DEPENDS:=glib2
+PKG_INSTALL:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+MAKE_INSTALL_FLAGS += prefix=/usr libdir=/usr/lib
+
+define Package/quasselc
+    SECTION:=libs
+    CATEGORY:=Libraries
+    DEPENDS:=+glib2
+    SUBMENU:=Instant Messaging
+    URL:=https://github.com/phhusson/QuasselC
+    TITLE:=API to access a Quassel Core in pure C
+endef
+
+define Package/quasselc/description
+  An implementation of the Quassel protocol in pure C.
+endef
+
+define Build/InstallDev
+	$(INSTALL_DIR) $(1)/usr/include/quasselc
+	$(CP) $(PKG_INSTALL_DIR)/usr/include/quasselc/* $(1)/usr/include/quasselc/
+
+	$(INSTALL_DIR) $(1)/usr/lib
+	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libquasselc.so* $(1)/usr/lib/
+	$(LN) -s libquasselc.so.0 $(1)/usr/lib/libquasselc.so
+
+	$(INSTALL_DIR) $(1)/usr/lib/pkgconfig
+	$(CP) $(PKG_INSTALL_DIR)/usr/lib/pkgconfig/quasselc.pc $(1)/usr/lib/pkgconfig/
+endef
+
+define Build/Compile
+	+$(MAKE_VARS) \
+	$(MAKE) $(PKG_JOBS) -C $(PKG_BUILD_DIR)/$(MAKE_PATH) \
+		prefix=$(STAGING_DIR)/usr libdir=$(STAGING_DIR)/usr/lib includedir=$(STAGING_DIR)/usr/include $(MAKE_FLAGS);
+endef
```

Comment:
We can avoid redefining `Build/Compile` by appending these additional command line variables to `MAKE_FLAGS`, like what has already be done with `MAKE_INSTALL_FLAGS` above.